csm_web/frontend/src/css/profile.scss

```diff
@@ -0,0 +1,224 @@
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  width: 100%;
+}
+
+.user-profile-image-container {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+  width: 8em;
+  height: 8em;
+  overflow: hidden;
+}
+
+.user-profile-border {
+  border: 2px dashed #ccc;
+  border-radius: 12px;
+}
+
+.user-profile-image {
+  width: 100%;
+}
+
+.user-profile-placeholder {
+  width: 80%;
+}
+
+.user-profile-fields {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  word-break: break-word;
+  overflow-wrap: break-word;
+  white-space: normal;
+}
+
+.user-profile-item {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+
+.user-profile-text {
+  margin: 0;
+}
+
+.user-profile-name {
+  font-size: 1.5rem;
+  color: $csm-green-darkened;
+}
+
+.user-profile-pronunciation {
+  font-style: italic;
+  color: $csm-neutral-darkened;
+}
+
+.user-profile-email {
+  color: $csm-neutral-darkened;
+}
+
+.user-profile-bio {
+  display: flex;
+  gap: 1rem;
+  justify-content: left;
+  width: 100%;
+  word-break: break-word;
+
+  overflow-wrap: break-word;
+  white-space: normal;
+}
+
+.user-profile-tooltip {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: auto;
+  height: auto;
+
+  .icon {
+    width: 1.25rem;
+    height: 1.25rem;
+  }
+}
+
+.user-tooltip-icon {
+  color: $csm-neutral;
+}
+
+.user-profile-input {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  align-items: center;
+  width: 100%;
+}
+
+.user-profile-image-upload-container {
+  position: relative;
+  display: inline-block;
+}
+
+.user-profile-image-upload {
+  display: none;
+}
+
+.user-profile-image-upload-label {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.user-profile-image-upload-icon {
+  position: absolute;
+  inset: 0;
+  width: 75%;
+  margin: auto;
+  opacity: 0.5;
+}
+
+.user-profile-image-upload-label:hover .user-profile-image-upload-icon {
+  opacity: 0.75;
+}
+
+.user-profile-clear-image-button {
+  position: absolute;
+  top: 0.25rem;
+  right: 0.25rem;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: $csm-danger;
+  cursor: pointer;
+
+  background-color: transparent;
+  border: none;
+  outline: none;
+
+  &:hover {
+    color: $csm-danger-darkened;
+  }
+
+  .icon {
+    width: 1.1rem;
+    height: 1.1rem;
+  }
+}
+
+.user-profile-name-label {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  gap: 0.2rem;
+  align-items: center;
+}
+
+.user-profile-preferred-name-tooltip {
+  flex: 0;
+}
+
+.user-profile-input-field {
+  width: 100%;
+}
+
+.user-profile-inline {
+  display: inline;
+}
+
+input[disabled] {
+  color: $csm-neutral-darkened;
+}
+
+.user-profile-bio-container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.2rem;
+  align-items: flex-start;
+  width: 100%;
+}
```

Comment:
There could probably be a little more space before the bio container; visually seems a little too close to the email input

<img width="607" height="578" alt="image" src="https://github.com/user-attachments/assets/485e60ce-8f48-41b9-9fbd-cae596280c5b" />